src/datasets/providers/flipper.ts

```diff
@@ -0,0 +1,15 @@
+import FlipperNoFillSvg from '@site/static/img/flipper-no-fill.svg';
+import { Provider } from '.';
+
+export const Flipper: Provider = {
+  name: 'Flipper',
+  logo: FlipperNoFillSvg,
+  technologies: [
+    {
+      technology: 'Ruby',
+      vendorOfficial: true,
+      href: 'https://github.com/flippercloud/flipper',
```

Comment:
Can you link to a section that talks about the OpenFeature provider for Flipper?